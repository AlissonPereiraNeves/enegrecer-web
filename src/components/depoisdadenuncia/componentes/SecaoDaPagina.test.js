import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SecaoDaPagina from './SecaoDaPagina';

describe('Secao da pagina depois da denuncia', () => {
  const wrapper = shallow(<SecaoDaPagina
    nomeDaClasse="classe-teste"
    textoDoBloco="texto-teste"
    imagemSecao="imagem-teste"
    ladoTexto="lado-teste"
    nomeFoto="nome-teste"
  />);

  it('possui a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
