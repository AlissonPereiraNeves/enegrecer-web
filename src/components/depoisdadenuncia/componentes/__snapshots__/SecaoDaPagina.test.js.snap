// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secao da pagina depois da denuncia possui a mesma estrutura 1`] = `
<div
  className="row depois_denuncia_secao"
>
  <div
    className="col s12 m6 conteudo_lado-teste"
  >
    <div
      className="detalhe_titulo"
    >
      <h1 />
      <div
        className=""
      />
    </div>
    <h1 />
    <div
      className="texto_secao"
    >
      <p>
        texto-teste
      </p>
    </div>
  </div>
  <div
    className="col s12 m6 imagem_secao"
  >
    <img
      alt="Imagem de nome-teste"
      src="imagem-teste"
    />
  </div>
</div>
`;
